main.tex

```diff
@@ -87,7 +93,7 @@ \subsection{Test files}
 
 A \emph{test file} consists of a) \TeX{} code that configures the Markdown package, b) markdown text, and c)~the expected output in the \texttt{.log} file.
 
-An example test file \file{strike-through.test} tests the strike-through syntax extension:
+As an example, \file{strike-through.test} tests the strike-through syntax extension:
```

Comment:
I assume that the change was made to improve the line breaking after the removal of inline icons. However, since we wish to keep the inline icons, this change seems unnecessary and inconsistent with other similar examples throughout the text.

```suggestion
An example test file \file{strike-through.test} tests the strike-through syntax extension:
```